src/daemon/command_server.cpp

```diff
@@ -362,7 +362,12 @@ t_command_server::t_command_server(
     , "print_checkpoints [+json] [start height] [end height]"
     , "Query the available checkpoints between the range, omit arguments to print the last 60 checkpoints"
     );
-
+    m_command_lookup.set_handler(
+      "print_sn_state_changes"
+    , std::bind(&t_command_parser_executor::print_sn_state_changes, &m_parser, p::_1)
+    , "print_sn_state_changes start height [end height]"
```

Comment:
`print_sn_state_changes <start height> [end height]`

start height is mandatory so <> notation

Reply:
Changed.